business/istio_config.go

```diff
@@ -193,7 +125,7 @@ func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (
 			if kialiCache != nil && kialiCache.CheckIstioResource(kubernetes.GatewayType) && kialiCache.CheckNamespace(criteria.Namespace) {
 				gg, ggErr = kialiCache.GetIstioResources(kubernetes.GatewayType, criteria.Namespace)
 			} else {
-				gg, ggErr = in.k8s.GetGateways(criteria.Namespace)
+				gg, ggErr = in.k8s.GetIstioObjects(criteria.Namespace, kubernetes.Gateways, "")
```

Comment:
Some day golang will reach modern age and have generics :-)

Reply:
Yeap, there are missing features in Golang I miss, too, like to differentiate signatures by arguments, etc.